R/onLoad.R

```diff
@@ -0,0 +1,5 @@
+.onLoad <- function(libname, pkgname) {
+  if (is.null(getOption("rsconnect.max.bundle.size"))) {
+    options(rsconnect.max.bundle.size = 1048576000)
```

Comment:
It is the size limit of the payload, i.e. we check the `Content-Length` header.

Practically speaking the 5 GB limit is the threshold at which you cease being able to upload to S3 in a single PUT call.

We probably should switch to multi-part for everything, per their recommendation. But really what I would like to do is supply `rsconnect` with a signed S3 URL and not have shinyapps.io in the middle at all.

https://docs.aws.amazon.com/AmazonS3/latest/dev/UploadingObjects.html